the_mines/get_html.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import shutil
+import os
+
+
+def get_html(matchday, season):
```

Comment:
Funny things can happen when importing a method that is the same name as the module.  I would advise changed either the module name or the method name.